engine/language-server/src/main/scala/org/enso/languageserver/boot/MainModule.scala

```diff
@@ -308,6 +308,7 @@ class MainModule(serverConfig: LanguageServerConfig, logLevel: Level) {
   val extraOptions = new java.util.HashMap[String, String]()
   extraOptions.put(RuntimeServerInfo.ENABLE_OPTION, "true")
   extraOptions.put(RuntimeOptions.INTERACTIVE_MODE, "true")
+  extraOptions.put(RuntimeOptions.DISABLE_LINTING, "true")
```

Comment:
Why don't you use `ContextFactory.disableLinting(true)`?

Reply:
sure